build.gradle

```diff
@@ -59,6 +64,9 @@ dependencies {
     implementation "javax.xml.bind:jaxb-api:2.3.1"
     implementation "com.guicedee.services:guice:${guicedeeVersion}"
     implementation "org.lz4:lz4-java:${lz4Version}"
+    implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonYamlVersion}"
+    implementation "com.fasterxml.jackson.dataformat:jackson-dataformat-yaml:${jacksonYamlVersion}"
```

Comment:
Are the jackson dependencies required? I think log4j2 can read the configs without it

Reply:
Yup, they are required: https://logging.apache.org/log4j/2.x/runtime-dependencies.html (search for YAML) Without this package log4j2 is unable to read yaml config file